jdaviz/configs/rampviz/plugins/parsers.py

```diff
@@ -196,6 +196,12 @@ def _roman_3d_to_glue_data(
     data_reshaped = move_group_axis_last(data)
     diff_data_reshaped = move_group_axis_last(diff_data)
 
+    # if the ramp cube has no units, assume DN:
+    if getattr(data_reshaped, 'unit', None) is None:
```

Comment:
Is there a reason to not use `hasattr`?
```suggestion
    if not hasattr(data_reshaped, 'unit'):
```

Reply:
`data_reshaped` could be an `NDData` or `Quantity` with an attr `unit = None`. This approach catches that case.